Latex/metadaten.sty

```diff
@@ -1,22 +1,26 @@
 %Autor 1:
 \newcommand{\autoreins}{Name 1}
 %Autor 2:
-\newcommand{\autorzwei}{Name 2}
+\newcommand{\autorzwei}{}
 %Autor 3:
-\newcommand{\autordrei}{Name 3}
-%Autoren - für PDF Titel
-\newcommand{\autoren}{{\autoreins}, {\autorzwei}, \autordrei}
+\newcommand{\autordrei}{}
+%Autor 4:
+\newcommand{\autorvier}{}
+%Autoren - für PDF Titel - anpassen, falls nicht alle genutzt werden
+\newcommand{\autoren}{{\autoreins}, {\autorzwei}, {\autordrei}, \autorvier}
```

Comment:
das hat sicher nen sinn, weil das vorher auch so war, aber: warum keine `{}` um autor 4?
und: die idee mit dem leer lassen ist zwar schön, aber dann steht halt da `Max Mustermann,,,`. Nicht optimal und eventuell leichter zu übersehen als `Max Musermann, Name 2, Name3`

Reply:
```
\newcommand{\checkfor}[1]{%
  \ifcsname#1\endcsname%
    ... command '#1' exists ...%
  \else%
    ... command '#1' does not exist ...%
  \fi%
}
```

Reply:
sehr gut, mach mal xD klingt ja recht simpel

Reply:
tracken in #128 und hier vorerst ignorieren